tyrian/js/src/main/scala/tyrian/TyrianAppF.scala

```diff
@@ -42,29 +43,56 @@ trait TyrianAppF[F[_]: Async, Msg, Model]:
     */
   @JSExport
   def launch(containerId: String): Unit =
-    ready(containerId, Map[String, String]())
+    runReadyOrError(containerId, Map[String, String]())
+
+  /** Launch the app and attach it to the given element. Can be called from Scala or JavaScript.
+    */
+  @JSExport
+  def launch(node: Element): Unit =
+    ready(node, Map[String, String]())
 
   /** Launch the app and attach it to an element with the given id, with the supplied simple flags. Can be called from
     * Scala or JavaScript.
     */
   @JSExport
   def launch(containerId: String, flags: scala.scalajs.js.Dictionary[String]): Unit =
-    ready(containerId, flags.toMap)
+    runReadyOrError(containerId, flags.toMap)
+
+  /** Launch the app and attach it to the given element, with the supplied simple flags. Can be called from Scala or
+    * JavaScript.
+    */
+  @JSExport
+  def launch(node: Element, flags: scala.scalajs.js.Dictionary[String]): Unit =
+    ready(node, flags.toMap)
 
   /** Launch the app and attach it to an element with the given id, with the supplied simple flags. Can only be called
     * from Scala.
     */
   def launch(containerId: String, flags: Map[String, String]): Unit =
-    ready(containerId, flags)
+    runReadyOrError(containerId, flags)
+
+  /** Launch the app and attach it to the given element, with the supplied simple flags. Can only be called from Scala.
+    */
+  def launch(node: Element, flags: Map[String, String]): Unit =
+    ready(node, flags)
 
-  def ready(parentElementId: String, flags: Map[String, String]): Unit =
+  def ready(node: Element, flags: Map[String, String]): Unit =
     run(
       Tyrian.start(
-        document.getElementById(parentElementId),
+        node,
         init(flags),
         update,
         view,
         subscriptions,
         MaxConcurrentTasks
       )
     )
+
+  @SuppressWarnings(Array("scalafix:DisableSyntax.throw"))
+  private def runReadyOrError(containerId: String, flags: Map[String, String]): Unit =
+    Option(document.getElementById(containerId)) match
+      case Some(e) =>
+        ready(e, Map[String, String]())
```

Comment:
Don't we want to pass `flags` to `ready` ? 👀

Reply:
Ooops! Thank you. I guess that's why I code in the evening and review in the morning. 😄 
Will fix this evening before I merge it in. I do miss Scala 2's 'unused' compiler warnings at times like this...